### ISSUE TITLE
[Request] Clearlogo on Fullscreen Widget

### ISSUE THREAD

**Issue by drinfernoo** [jurialmunkey/skin.arctic.horizon#269]
First of all, I could've *sworn* that either the Showcase Hub or the screensaver used to show clearlogo in this position... but It would be really sweet if we could get clearlogo in the bottom right of these "fullscreen"-styled widget scenarios. I made a quick mockup, as an example, but it's certainly possible my measurements aren't quite right :sweat_smile:

![clearlogo](https://user-images.githubusercontent.com/2319508/91925377-b59f1d80-ec89-11ea-9ae8-d19c76ab4439.png)

**Comment by SerpentDrago** [comment]
I think it would be cool if it would Replace the text if available , and fall back to text if not

**Comment by SerpentDrago** [comment]
like this > 

![image](https://user-images.githubusercontent.com/21133858/91926564-826a8780-eca5-11ea-9c5f-a40808200ed4.png)

**Comment by SerpentDrago** [comment]
Honestly while we are at it , offer option to remove the top left "section" label , where it would have said power for me 

![image](https://user-images.githubusercontent.com/21133858/91927170-d0cc5600-eca6-11ea-8488-182c9290ed97.png)

**Comment by drinfernoo** [comment]
@SerpentDrago That one is coming: https://github.com/jurialmunkey/skin.arctic.horizon/issues/243

But I think the main reason he's always stayed away from having such a prominent clearlogo is due to what are often drastic differences in sizing and relative position, as well as availability.

I do agree it looks nice, but also feel clearlogo along the right side of the screen "lines up" *better* visually with clearart in the skin.

**Comment by SerpentDrago** [comment]
Yeh i can understand the reason not to . i'd just like the option even if its not always perfect depending on the logo . I think the "fall back title text" could be a bit larger to . 

great to hear about the header text !

**Comment by jurialmunkey** [comment]
@SerpentDrago - Pretty much exactly what drinfernoo said. The main reason I've shied away from using clearlogos as title is because they don't have strong alignment edge (unlike clearart which aligns bottom right) and will often have blank/dead transparent space with a centered logo. The [top rated Jurassic Park logo](https://fanart.tv/fanart/movies/329/hdmovielogo/jurassic-park-505a682b2dffc.png) particularly annoys me, particularly since there is [another version](https://fanart.tv/fanart/movies/329/hdmovielogo/jurassic-park-548f54037279f.png) which uses the space much better.

Right or center positioning for clearlogo works because exact alignment isn't as important since there isn't text underneath (an uneven margin makes your eyes work much harder to parse text quickly on screen).

Also, making the clearlogo that large is a problem because it won't align correctly with the top edge when you scroll to the next widget. Having clearlogo there means forgoing either ratings, plot or buttons -- all of which I imagine are probably more highly preferred: ratings/plot because that widget is typically about discovery of something to watch; buttons because people with six button remotes don't have a way to access info without context menu and there's no "<oncontext>" tag that will allow me to pass the context action through to the background list.

@drinfernoo - to answer your question, yes fullscreen widget definitely had clearlogo as seen in [this screenshot](https://raw.githubusercontent.com/jurialmunkey/skin.arctic.horizon/master/resources/screenshot-03.jpg) - just need to figure a few things out in terms of getting it to work properly for both menu types. Problem essentially boils down to figuring out a way to hide it when second widget is focused for horizontal menu. The vert menu is fine because it will hide whenever fullscreen widget isn't focused since it needs to hide for the slide over animation. However, since horizontal doesn't slide, the  clearlogo needs to be visible anytime fullscreen widget is onscreen but needs to hide when second widget is focused because of the clearart. Problem is that getting details about a different widget in a skinshortcut template is difficult.

**Comment by jurialmunkey** [comment]
Fixed in latest.

**Comment by SerpentDrago** [comment]
@jurialmunkey  I understand your reasons . It is jarring when the logo doesn't really align , and fallback text can be jarring . 

Just fyi if you never seen Auramod's showcase . 

https://youtu.be/OGNfF37EZls

Also if you could look at that  for another thing i wanted to run past you . While i have you here . The Catagory widgets underneath the showcase . Is there any way you can add the ability to set background in AH kinda like they are in auramod . 

Basically in auramod you can set background on the category widgets , Then you set the icon to "none" icon , and it will then center the Widget name . I think it could look fantastic in AH . with Ah's nice smooth fonts and borders . I'd kill to have this ability without having to fork it myself . 

Anyways , just a suggestion i'll leave you be for now ;0

**Comment by jurialmunkey** [comment]
@SerpentDrago 

**Submenu Backgrounds (1)**
I had actually been playing around with a similar idea quite a while ago where I wanted to include backgrounds not just for submenu items but also for library nodes - I never got around to fully implementing it because I ran into the issue of what to do with custom nodes/playlists etc. Plus it was a lot of work sourcing suitable and nice looking background artwork that also had an appropriate license. I wish I saved some screenshots because it looked quite good.

Opened an issue here: https://github.com/jurialmunkey/skin.arctic.horizon/issues/270

**Item Indicator (2)**
Yep good idea, I'll look at adding something to indicate the item position for background widget. The main issue is that because the item limit can be variable, then something like dots are not really feasible and I'd rather not have yet another piece of text floating somewhere on screen. I'll see what I can come up with.

Opened an issue here: https://github.com/jurialmunkey/skin.arctic.horizon/issues/271

**Widget Alignment (3)**
Honestly alignment is already a pain as it is -- other than the fact that Kodi doesn't really give any control over positioning in grouplists and so requires a series of hacks to force the positioning I want, the big issue is that skinshortcut templates don't really have a way for each widget to get info about other widgets in the group

e.g. It's pretty much impossible to tell what the widget above/below will be and then include the correct top/height positioning in the template on that basis. Instead I have to resort to less than satisfactory hacks/tricks that involve reactionary visibility conditions. That's really limiting in terms of variations as each potential configuration then requires a bunch of hidden controls which become visible according to configuration rather than only including what is needed (and more controls means worse performance and more chance of bugs/unexpected behaviour).

I think the solution here is probably more likely having a similar type of "lovefilm / large landscape showcase" widget as shown in the video of auramod where the height makes it fit properly on-screen with the submenu. I'll have a think about this and see what I come up with

**Comment by jurialmunkey** [comment]
Also on the topic of clearlogo as title replacement, here are a few examples of why it doesn't work. These are very popular shows using the default highest rated artwork - so it's not like I went to trouble to find bad examples.

![Honeyview_screenshot209](https://user-images.githubusercontent.com/4618719/92080638-37678600-ee05-11ea-80c3-b8a66aeb78c5.jpg)
![Honeyview_screenshot212](https://user-images.githubusercontent.com/4618719/92080648-3b93a380-ee05-11ea-9733-4f530f0dcbce.jpg)

Even with logos that are more suitable, there's often vertical spacing issues due to dead space:
![Honeyview_screenshot218](https://user-images.githubusercontent.com/4618719/92080720-52d29100-ee05-11ea-9ec2-76b5bf9c5d6d.jpg)


Of course, with the correct cropping the logos can look amazing. Yet even with careful selection of the logos I'm using, I still needed to download and crop all of these manually to get them to align nicely.

![Honeyview_screenshot213](https://user-images.githubusercontent.com/4618719/92080840-80b7d580-ee05-11ea-8bf7-25838a7fed5b.jpg)
![Honeyview_screenshot215](https://user-images.githubusercontent.com/4618719/92080844-83b2c600-ee05-11ea-94ee-e45342db13e1.jpg)
![Honeyview_screenshot216](https://user-images.githubusercontent.com/4618719/92080853-87464d00-ee05-11ea-8304-b9628e957298.jpg)
![Honeyview_screenshot214](https://user-images.githubusercontent.com/4618719/92080858-88777a00-ee05-11ea-8052-f662daec74ab.jpg)

**Comment by SerpentDrago** [comment]
> I think the solution here is probably more likely having a similar type of "lovefilm / large landscape showcase" widget as shown in the video of auramod where the height makes it fit properly on-screen with the submenu. I'll have a think about this and see what I come up with

Yes , i like that alot ! 
![image](https://user-images.githubusercontent.com/21133858/92114507-ee8cde80-edbe-11ea-9c14-f89daf43c58a.png)

**Comment by SerpentDrago** [comment]
Thanks for the response's
